.github/workflows/pr-workflow.yml

```
@@ -125,25 +150,47 @@
           make docker.build
 
+          # Save image to cache
+          docker save $ECR_NAME:$COMMIT | gzip > /tmp/main-api-image.tar.gz
+          echo "image=$ECR_NAME:$COMMIT >> $GITHUB_OUTPUT
+
+      - name: Cache Docker image
+        uses: actions/cache@v4
+        with:
+          path: /tmp/main-api-image.tar.gz
+          key: main-api-image-${{ github.sha }}
+          retention-days: 7
```

actionlint (1.7.7)

input "retention-days" is not defined in action "actions/cache@v4". available inputs are "enableCrossOsArchive", "fail-on-cache-miss", "key", "lookup-only", "path", "restore-keys", "save-always", "upload-chunk-size"

(action)